udata_ckan/harvesters.py

```diff
@@ -101,6 +101,11 @@
 
 class CkanBackend(BaseBackend):
     display_name = 'CKAN'
+    filters = (
+        HarvestFilter(_('Organization'), 'organization', str,
+                      _('A CKAN Organization name')),
+        HarvestFilter(_('Tag'), 'tags', str, _('A CKAN tag name')),
```

Comment:
Why `tags` (plural)? CKAN API weirdness?

Reply:
Yes, one of the CKAN API weirdness :unamused: